lib/internal/modules/cjs/loader.js

```diff
@@ -209,12 +238,7 @@ function internalRequire(module, id) {
     throw new ERR_INVALID_ARG_VALUE('id', id,
                                     'must be a non-empty string');
   }
-  requireDepth++;
-  try {
-    return Module._load(id, module, /* isMain */ false);
-  } finally {
-    requireDepth--;
-  }
+  return wrapModuleLoad(id, module, /** isMain */ false);
```

Comment:
This should still keep the original try-finally with just the  `Module._load` replaced with `wrapModuleLoad()`, and only do the requireDepth computation in `internalRequire` and `Module.prototype.require`, so that other code that does `Module._load` can also just replace it with `wrapModuleLoad()`